### PR TITLE
Feature/fpc 1100 display message unkown rfid tag instead of missing entitlement

### PR DESCRIPTION
**Pull request by f-richter**
- add missing entitlement message
- set success message for successful reading tags without ndef support

### CODE REVIEW

File: src/Plugin.NFC/iOS/NFC.iOS.cs [hunk 1]
```diff
@@ -205,6 +207,8 @@ public override void DidDetectTags(NFCTagReaderSession session, INFCTag[] tags)
 						// let the user decide if ndef support is needed
 						OnMessageReceived?.Invoke(nTag);
 
+						session.AlertMessage = Configuration.Messages.NFCSuccessRead;
```
Comment (convivo) [inline src/Plugin.NFC/iOS/NFC.iOS.cs:210]: Is this need or was this just for debugging?

Comment (f-richter) [reply src/Plugin.NFC/iOS/NFC.iOS.cs:210]: This is necessary for sub task FPC-1106

File: src/Plugin.NFC/Plugin.NFC.csproj [hunk 2]
```diff
@@ -1,4 +1,4 @@
-﻿<?xml version="1.0" encoding="UTF-8"?>
+﻿<?xml version="1.0" encoding="utf-8"?>
```
Comment (convivo) [inline src/Plugin.NFC/Plugin.NFC.csproj]: Please revert this change

Comment (f-richter) [reply src/Plugin.NFC/Plugin.NFC.csproj]: Done
